Adding method in API to list all versions of an entity (#114)
HPInc/ml-git: docs/api/README.md

        Returns:
            list of class Entity.
        """
    def get_entity_versions(self, entity_name, metadata_repo_name):
        """Get a list of spec versions found for an especific entity.

        Args:
            entity_name (str): The name of the entity you want to get the versions.

>             metadata_repo_name (str): The repository name where the metadata is located in GitHub.

Reviewer: The repository name of config,yaml or a repository of versioned entitiy?

Author: The message has been updated to make it clearer which repository.
Also, some variables have been renamed, so that the user can understand when to use the configuration repository (`config_repo_name`) and when to use the metadata repository (`metadata_repo_name`).

These changes were applied at d364d49.

```python
class Entity:
    """Class that represents an ml-entity.

    Attributes:
        name (str): The name of the entity.
        entity_type (str): The type of the ml-entity (datasets, models, labels).
        private (str): The access of entity metadata.
        metadata (Metadata): The metadata of the entity.
        last_spec_version (SpecVersion): The spec file of entity last version.
    """
```
</details>

<details markdown="1">
<summary><code> SpecVersion </code></summary>
<br>

```python
class SpecVersion:
    """Class that represents an ml-entity spec version.

    Attributes:
        name (str): The name of the entity.
        entity_type (str): The type of the ml-entity (datasets, models, labels).
        version (str): The version of the ml-entity.
        tag (str): The tag of the ml-entity spec version.
        mutability (str): The mutability of the ml-entity.
        categories (list): Labels to categorize the entity.
        storage (Storage): The storage of the ml-entity.

>         amount (str): The amount of the version files.

Author: The suggestion was applied in 06ec778.

    """Class that's represents a ml-entity.

    Attributes:
        name (str): The name of the entity.
        entity_type (str): The type of the ml-entity (datasets, models, labels).
        private (str): The access of entity metadata.
        metadata_full_name (str): The name of the repository metadata.
        metadata_git_url (str): The git url of the repository metadata.
        metadata_html_url (str): The html url of the repository metadata.
        metadata_owner_name (str): The name of the repository owner.

>         metadata_owner_email (str): The email of the repository owner.

Author: The changes that had been applied in #115, related to the metadata and storage models, were brought to this branch.

>     """Class that's represents a ml-entity.

Reviewer 2: ```suggestion
    """Class that represents an ml-entity.
```

Author: Text has been updated in c8ce0bd.

    Attributes:
        name (str): The name of the entity.
        entity_type (str): The type of the ml-entity (datasets, models, labels).
        private (str): The access of entity metadata.
        metadata (Metadata): The metadata of the entity.
        last_spec_version (SpecVersion): The spec file of entity last version.
    """
```
</details>

<details markdown="1">
<summary><code> SpecVersion </code></summary>
<br>

```python
class SpecVersion:

>     """Class that's represents a ml-entity spec version.

Reviewer 2: ```suggestion
    """Class that represents an ml-entity spec version.
```

Author: Text has been updated in c8ce0bd.

    Attributes:
        name (str): The name of the entity.
        entity_type (str): The type of the ml-entity (datasets, models, labels).
        version (str): The version of the ml-entity.
        tag (str): The tag of the ml-entity spec version.
        mutability (str): The mutability of the ml-entity.
        categories (list): Labels to categorize the entity.
        storage (Storage): The storage of the ml-entity.
        amount (str): The amount of the version files.
        size (str): The size of the version files.
    """
```
</details>

<details markdown="1">
<summary><code> Metadata </code></summary>
<br>

```python
class Metadata:

>     """Class that's represents an ml-entity metadata.

Reviewer 2: ```suggestion
    """Class that represents an ml-entity metadata.
```

Author: Text has been updated in c8ce0bd.

    Attributes:
        full_name (str): The full name of the metadata.
        git_url (str): The git url of the metadata.
        html_url (str): The html url of the metadata.
        owner_email (str): The owner email of the ml-entity metadata.
        owner_name (str): The owner name of the ml-entity metadata.
    """
```
</details>

<details markdown="1">
<summary><code> Storage </code></summary>
<br>

```python
class Storage:

>     """Class that's represents an ml-entity storage.

Reviewer 2: ```suggestion
    """Class that represents an ml-entity storage.
```

Author: Text has been updated in c8ce0bd.
